meta-intel-edison-distro/recipes-core/initrdscripts/initramfs-framework_1.0.bbappend

```diff
@@ -1,6 +1,6 @@
 SUMMARY = "Live image init script"
 #LIC_FILES_CHKSUM = "file://${COREBASE}/meta/COPYING.MIT;md5=3da9cfbcb788c80a0384361b4de20420"
 
-FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
-SRC_URI += "file://init-live.sh \
+FILESEXTRAPATHS_prepend := "${THISDIR}/initramfs-framework:"
+SRC_URI += "file://init \
```

Comment:
This one isn;'t really necessary - the main recipe has this already, so it's enough to just prepend our directory into the `FILESEXTRAPATHS` and it will get picked up.